### ISSUE TITLE
Build failing due to transitive dependency on bintray

### ISSUE THREAD

**Issue by vincentml** [digital-preservation/csv-validator#308]
A Java Maven project that uses csv-validator has started failing builds, and it appears that the cause is some dependency within csv-validator relies on bintray.com which is now offline according to a message at https://bintray.com/. 

The error message in the build log is as follows:

Failed to collect dependencies at uk.gov.nationalarchives:csv-validator-java-api:jar:1.1 -> uk.gov.nationalarchives:csv-validator-core:jar:1.1 -> org.scalaz.stream:scalaz-stream_2.11:jar:0.6a: Failed to read artifact descriptor for org.scalaz.stream:scalaz-stream_2.11:jar:0.6a: Could not transfer artifact org.scalaz.stream:scalaz-stream_2.11:pom:0.6a from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [scalaz-bintray (http://dl.bintray.com/scalaz/releases, default, releases+snapshots), gilt-group (http://dl.bintray.com/giltgroupe/maven, default, releases+snapshots)] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

Is it possible to update the dependencies to pull from Maven Central?

**Comment by vincentml** [comment]
I have narrowed this problem down to a specific jar and found a workaround. I'm using csv-validator version 1.1.5, and in csv-validator-core/pom.xml there is an import for gfc-semver_2.11 version 0.1.0 which is not available in Maven Central. An earlier version 0.0.5 is available. 

The workaround was to change the pom.xml of my project to import gfc-semver_2.11 version 0.0.5 as follows:

```xml
<dependency>
	<groupId>uk.gov.nationalarchives</groupId>
	<artifactId>csv-validator-java-api</artifactId>
	<version>1.1.5</version>
	<exclusions>
		<exclusion>
			<groupId>com.gilt</groupId>
			<artifactId>gfc-semver_2.11</artifactId>
		</exclusion>
	</exclusions>
</dependency>
<dependency>
	<!-- csv-validator imports gfc-semver_2.11 version 0.1.0 which is not available in Maven Central, so import next best version that is available -->
	<groupId>com.gilt</groupId>
	<artifactId>gfc-semver_2.11</artifactId>
	<version>0.0.5</version>
</dependency>
```

It looks like csv-validator-core/pom.xml has already been updated to use gfc-semver version 0.0.5 for the next release. When will the next version of csv-validator be released?

**Comment by vincentml** [comment]
This problem and workaround is the same as issue #284

**Comment by adamretter** [comment]
This is fixed in f7c3878

> When will the next version of csv-validator be released?

@DavidUnderdown would you mind if I released a 1.2-RC3 version? After which we pause a little, and if no problems, are reported we release the final 1.2 version?

**Comment by DavidUnderdown** [comment]
The main reason we were keeping the 1.2 series as release candidates only is that we don't have a finalised v1.2 of CSV schema language. Ideally we'd do a 1.1.x release too as CSV Schema Language v1.1 is definitive.

**Comment by adamretter** [comment]
@DavidUnderdown Okay understood. I think doing a 1.1.x is difficult as the version of Scala etc has moved on and I think the code now incorporates some concepts from Schema 1.2.

Is it okay to do a 1.2-RC3?

**Comment by DavidUnderdown** [comment]
Yes, that's fine. We'll have to have a think about 1.1.x then too.

**Comment by adamretter** [comment]
> We'll have to have a think about 1.1.x then too

Unfortunately I think that would be quite tricky as the main line of development already contains some 1.2 features. So you would have to cherry-pick and build a new branch.

**Comment by adamretter** [comment]
@vincentml @DavidUnderdown I just released version 1.2-RC3 to Maven Central. It should be publicly available within the next two hours (their synchronisation window)
